.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -121,3 +121,22 @@ body:
       placeholder: v1.24.1
     validations:
       required: true
+
+  - type: input
+    id: postgresql_version
+    attributes:
+      label: PostgreSQL version
+      description: What version of PostgreSQL are you running? (see database -> image section in your `docker-compose.yml`)
+      placeholder: v17
+
+  - type: checkboxes
+    attributes:
+      label: Are you running latest PostgreSQL version?
+      description: |
+        Please search check with (installation docs)[https://docs.teslamate.org/docs/installation/docker/] if you are running the latest supported PostgreSQL version.
```

Comment:
search or check, not both?

alternative:

Please ensure you are running the latest major supported PostgreSQL version by comparing your installation with the (installation docs)[https://docs.teslamate.org/docs/installation/docker/].